### ISSUE TITLE
Platform specific SASS are not respected when bundling with WebPack

### ISSUE THREAD

**Issue by NickIliev** [NativeScript/nativescript-dev-webpack#188]
When we have platform specific SASS files (e.g. `platform.android.scss` and `platform.ios.scss`) and we bundle our application the files are not respected and the styles are not applied.

Test application [here](https://github.com/NickIliev/NativeScript-Issues-2017/tree/master/test/drawer-test)
Steps to reproduce:
- `tns prepare`
- `npm run start-android-bundle`
- the styles from [this file](https://github.com/NickIliev/NativeScript-Issues-2017/blob/master/test/drawer-test/app/platform.android.scss) won't be applied to [this page](https://github.com/NickIliev/NativeScript-Issues-2017/blob/master/test/drawer-test/app/home/home.component.html#L17).

Reported via SB while using the SASS template from[ this branch.](https://github.com/NativeScript/template-drawer-navigation-ng/tree/vegeorgi/sass-fixes).

**Comment by SBats** [comment]
Hi and thank you for your work on this plugin!

I would like to know if someone from your team has already started working on it.
Otherwise, I would gladly try to have a look at it!
And in this case, would you have any specific advice or insight regarding this issue?

**Comment by zenz** [comment]
I also have this problem.

Such like using Groceries nativescript core template for example. If using webpack to make the app, it will lost all the css, and if I add { from: "**/*.css"} to CopyWebpackPlugin section, then the app can run properly under android but still have problem under iOS.

The bundle-config.js is like following:

```
if (global["TNS_WEBPACK"]) {
    require("tns-core-modules/bundle-entry-points");
    global.registerModule("views/login/login", function () { return require("./views/login/login"); });
    global.registerModule("views/list/list", function () { return require("./views/list/list"); });
    global.registerModule("views/register/register", function () { return require("./views/register/register"); });
    // register more application modules here following the example above
}
```

**Comment by sis0k0** [comment]
Should be fixed with v0.9.0. The test apps in the repo cover these scenarios. Ping @NickIliev

**Comment by NickIliev** [comment]
Issue resolved with version 0.9.0 of nativescript-dev-webpack. Tested with `demo/JavaScriptApp`

**Comment by NickIliev** [comment]
Resolved  in 0.9.0 - confirmed via the test application in `demo/JavaScriptApp`
